networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/peers/Eth2PeerManager.java

```diff
@@ -119,9 +127,11 @@ public static Eth2PeerManager create(
       final int peerRequestLimit,
       final Spec spec,
       final KZG kzg,
-      final DiscoveryNodeIdExtractor discoveryNodeIdExtractor) {
+      final DiscoveryNodeIdExtractor discoveryNodeIdExtractor,
+      final Optional<UInt64> custodyGroupCount) {
 
-    final MetadataMessagesFactory metadataMessagesFactory = new MetadataMessagesFactory();
+    // FIXME: we have no guarantee here that it's synced already
```

Comment:
are we keeping track of this?

Reply:
https://github.com/Consensys/teku/issues/9426